src/wallet/wallet2.cpp

```diff
@@ -9784,6 +9787,7 @@ std::vector<wallet2::pending_tx> wallet2::create_transactions_2(std::vector<cryp
           if (dsts.size() >= max_dsts)
             return false;
           dsts.push_back(de);
+          dsts_are_fee_subtractable.push_back(subtracting_fee);
```

Comment:
This might be a better design...

Reply:
Okay after looking at it, I think it's a better option to not do this since we would need to update all the code which uses `tx_destination_entry` to construct transactions, including the multisig ringct tx builder class.